test/gtest/unit_conv_solver_ConvOclDirectFwd.cpp

```diff
@@ -26,6 +26,8 @@
 
 #include "unit_conv_solver.hpp"
 
+#define WORKAROUND_MIOPEN_1392 (HIP_PACKAGE_VERSION_FLAT >= 6004000000)
```

Comment:
```suggestion
// Disable this solver due to random GPU memory access faults on gfx11 and gfx12
#define WORKAROUND_LWPMIOPEN_1392 (HIP_PACKAGE_VERSION_FLAT >= 6004000000)
```

See the first comment